src/main/groovy/com/vanniktech/android/junit/jacoco/GenerationPlugin.groovy

```diff
@@ -159,7 +159,7 @@ class GenerationPlugin implements Plugin<Project> {
                                       JacocoMerge mergeTask, JacocoReport mergedReportTask, final String taskName,
                                       final String jvmTestTaskName, final String instrumentationTestTaskName, final String sourceName,
                                       final String sourcePath, final String productFlavorName, final String buildTypeName) {
-        final def destinationDir
+        def destinationDir
```

Comment:
Why was the `final` changed?

Reply:
Because this is not a valid assignment starting from Gradle 5.
If I leave the `final` keyword there I get the following compile error: `164: The variable [destinationDir] is declared final but is reassigned`